src/DataFixtures/PhoneFixtures.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\Phone;
+use App\Entity\Screen;
+use App\Entity\Size;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+
+/**
+ * GuestFixtures.
+ *
+ * Fixture class to load 10 phones in database
+ */
+class PhoneFixtures extends Fixture
+{
+    /**
+     * parameterBag.
+     *
+     * @var ParameterBagInterface
+     */
+    private ParameterBagInterface $parameterBag;
+
+    /**
+     * __construct.
+     *
+     * @param ParameterBagInterface $parameterBag
+     */
+    public function __construct(ParameterBagInterface $parameterBag)
+    {
+        $this->parameterBag = $parameterBag;
+    }
+
+    /**
+     * load data fixtures (10 phones) with the passed EntityManager.
+     *
+     * @param ObjectManager $manager
+     */
+    public function load(ObjectManager $manager): void
+    {
+        // Recovery in $phones initial data list of 10 phones, contained in the phones.ini file at the root of the application.
+        $phones = parse_ini_file(
```

Comment:
Codacy found an issue: [The use of function parse_ini_file() is discouraged](https://app.codacy.com/gh/CarolineDirat/P7APIBileMo/pullRequest?prid=6304570)